doc/VitessOverview.md

```diff
@@ -0,0 +1,145 @@
+Vitess is a database solution for scaling MySQL. It's architected to run as
+effectively in a public or private cloud architecture as it does on dedicated
+hardware. It combines and extends many important MySQL features with the
+scalability of a NoSQL database. Vitess has been serving all YouTube database
+traffic since 2011.
+
+### Vitess on Kubernetes
+
+Kubernetes is an open-source orchestration system for Docker containers, and Vitess is the logical storage engine choice for Kubernetes users.
+
+Kubernetes handles scheduling onto nodes in a compute cluster, actively manages workloads on those nodes, and groups containers comprising an application for easy management and discovery. Using Kubernetes, you can easily create and manage a Vitess cluster, out of the box.
+
+## Comparisons to other storage options
+
+The following sections compare Vitess to two common alternatives, a vanilla MySQL implementation and a NoSQL implementation.
+
+### Vitess vs. Vanilla MySQL
+
+Vitess improves a vanilla MySQL implementation in several ways:
+
+<style>
+  table td {
```

Comment:
Thanks for noticing this. I removed the CSS from this file and stuck it in the extra.css file.